instructions.html

```diff
@@ -15,30 +16,19 @@
                     <div class="version">3.0</div>
                 </a>
                 <div class="section-grid">
-                    <h1 class="section-header">Speedometer 3 Test Instructions</h1>
+                    <h1 class="section-header">Speedometer 3 General Instructions</h1>
                     <div class="section-content">
-                        <h2>General Instructions</h2>
-                        <ol>
-                            <li>Before running Speedometer:</li>
-                            <ul>
-                                <li>Close all other applications.</li>
-                                <li>Close other browser windows and tabs.</li>
-                                <li>Use a fresh browser profile and a single window.</li>
-                            </ul>
-                            <li>While running Speedometer:</li>
-                            <ul>
-                                <li>Keep the browser window in the foreground.</li>
-                                <li>Do not use any other applications while Speedometer runs.</li>
-                            </ul>
-                        </ol>
-
-                        <!-- TODO: fill in detailed instructions
-                        <h2>Detailed browser-specific instructions</h2>
-                        <h3>Chrome</h3>
-                        <h3>Edge</h3>
-                        <h3>Firefox</h3>
-                        <h3>Safari</h3>
-                    --></div>
+                        <p>To get reasonably accurate results your device and browser need to be focused on the benchmark. Here are some basic instructions to allow that:</p>
+                        <ul class="compact">
+                            <li>Ensure you are running the latest stable browser version.</li>
+                            <li>Prefer using a separate clean browser profile — extensions and non-default browser settings can have a large impact on the score.</li>
+                            <li>Don't run any programs in the background.</li>
+                            <li>Restart the browser and close all other tabs and windows.</li>
+                            <li>Keep the Speedometer page focused and do not interact with the device while running the benchmark.</li>
+                            <li>Keep your mobile device plugged in to your charger.</li>
```

Comment:
I don't think this applies to just mobile devices like phones. Even laptops can have a variance in its performance if it's connected to AC vs on battery.

Reply:
Changed this to "battery-powered device" to be a bit more generic.